src/frontend/src/optimizer/plan_node/stream_hash_agg.rs

```diff
@@ -27,14 +27,23 @@ use crate::utils::{ColIndexMapping, ColIndexMappingRewriteExt};
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub struct StreamHashAgg {
     pub base: PlanBase,
-    /// an optional column index which is the vnode of each row computed by the input's consistent
-    /// hash distribution
-    vnode_col_idx: Option<usize>,
     logical: LogicalAgg,
+
+    /// An optional column index which is the vnode of each row computed by the input's consistent
+    /// hash distribution.
+    vnode_col_idx: Option<usize>,
+
+    /// The index of `count(*)` in `agg_calls`.
+    row_count_idx: usize,
 }
 
 impl StreamHashAgg {
-    pub fn new(logical: LogicalAgg, vnode_col_idx: Option<usize>) -> Self {
+    pub fn new(logical: LogicalAgg, vnode_col_idx: Option<usize>, row_count_idx: usize) -> Self {
+        debug_assert_eq!(
```

Comment:
This is lightweight and it should be okay to remove the `debug_`.

Reply:
Ok